gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -1044,12 +901,11 @@ public void onFailure(GoogleJsonError jsonError, HttpHeaders responseHeaders) {
    */
   // TODO(b/120887495): This @VisibleForTesting annotation was being ignored by prod code.
   // Please check that removing it is correct, and remove this comment along with it.
-  // @VisibleForTesting
+  @VisibleForTesting
```

Comment:
Should we also remove the TODO comment above this?

Reply:
Done, I think this was a false-positive.